src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.input {
+  border-radius: 4px;
+  box-shadow: 0 1px 10px rgba(90, 90, 90, 0.2);
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  display: block;
+  font-family: inherit;
+  outline: none;
+}
+
+.input--big {
+  background: url(./images/Search.svg) no-repeat;
+  background-position: 25px 24px;
+  background-size: 19px;
+  margin-top: 20px;
+  padding: 23px 62px;
+  font-size: 16px;
+}
+
+.input--small {
+  background: url(./images/Search.svg) no-repeat;
```

Comment:
This rule is duplicated in `input--big` selector, you should use `,` selector for grouping selectors